modules/core/src/main/java/org/apache/ignite/internal/processors/cache/distributed/dht/topology/PartitionsEvictManager.java

```diff
@@ -265,6 +322,46 @@ private void showProgress() {
         evictionGrps.forEach(GroupEvictionContext::awaitFinishAll);
     }
 
+    /**
+     * Getting a string representation of reason for eviction.
+     *
+     * @param reason Reason for eviction.
+     * @return String of reason for eviction.
+     * */
+    private String reason(byte reason) {
+        switch (reason) {
+            case EVICT_REASON_EVICTION:
+                return "eviction";
+
+            case EVICT_REASON_CLEARING:
+                return "clearing";
+
+            default:
+                return "reason_" + reason;
+        }
+    }
+
+    /**
+     * Creating a group partitions for reasons of eviction as a string.
+     *
+     * @param evictParts Partitions with a reason for eviction.
+     * @return String with group partitions for reasons of eviction.
+     */
+    private String partByReasonStr(Set<EvictPartition> evictParts) {
+        assert nonNull(evictParts);
```

Comment:
Please use `A.notNull` instead.

Reply:
As per the description from https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-Assertionsandparametervalidation
we must use assert.